src/web/lib/formatAttrString.ts

```diff
@@ -0,0 +1,28 @@
+import flow from 'lodash.flow';
+
+export const formatAttrString = (input: string) => {
+    // Remove all non alphaNumeric
+    // Lowercase all
+    // Replace Spaces with Hyphens
+
+    type strInOutFunc = (str: string) => string;
+
+    const removeAlphaNumeric: strInOutFunc = str =>
+        str.replace(new RegExp('[^0-9a-z ]', 'gi'), ';'); // We set it to semicolon so we can remove it at the end
+    const lowercase: strInOutFunc = str => str.toLowerCase();
+    const replaceSpacesWithHyphens: strInOutFunc = str =>
+        str.replace(new RegExp(' ', 'g'), '-'); // Replace the space
+    const removeTempChars: strInOutFunc = str =>
+        str.replace(new RegExp('[;]', 'g'), ''); // Strip out those semicolons
+
+    // https://lodash.com/docs/4.17.15#flow
```

Comment:
Lodash flow is 1kb: https://bundlephobia.com/result?p=lodash.flow@3.5.0 so I'm aware I'm in the realm of nitpicking but maybe more ugly code is worth slightly less bytes? I don't have strong feelings on this one though!

Reply:
Fair point.

There's a certain beauty in 

![image](https://user-images.githubusercontent.com/638051/75039938-a0169300-54b1-11ea-9491-042b68996244.png)

Reply:
Oh no... what have I done 🤦‍♂

Reply:
too late! merged!

![tenor-217053173](https://user-images.githubusercontent.com/638051/75040658-f6380600-54b2-11ea-893a-7a3ff0ef1bf5.gif)